profiles/sovereign36.md

```diff
@@ -0,0 +1,8 @@
+# Bohdan Feniak
+
+Ain't no picture here.
```

Comment:
You could still add `![](https://github.com/sovereign36.png)` and it'd show your profile image even if you set it later from https://github.com/settings!